### PR TITLE
Patricia Manalang- Completed html-forms exercise.

### PR DESCRIPTION
**Pull request by patriciamanalang**
Screen Shot 2022-10-11 at 11 34 53 AM

### CODE REVIEW

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey Patricia! You should have all of your form controls inside one `html` element, not a separate one for each form control

**Comment by patriciamanalang** [comment]
Screen Shot 2022-10-11 at 4 58 07 PM

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey Patricia, you are missing all `label` elements, please revise to include labels where necessary.

**Comment by patriciamanalang** [comment]
Screen Shot 2022-10-12 at 6 12 35 PM

File: html-forms/index.html [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  <h1>Coding Form!</h1>
+  <form>
+    <div>
+      <label>
+        Email<input type="text" name="email"/>
+      </label>
+    </div>
+    <div>
+      Why is coding cool?
+    </div>
```
Comment (codingcodymiller) [inline html-forms/index.html]: This should be a `label`

File: html-forms/index.html [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        <option value="Select an option" selected="selected">Select an option</option>
+      </select>
+    </label>
+
+
+    <p>Favorite Language (choose one)</p>
+    <label>
+      <input type="radio" name="languages" value="HTML"/>HTML
+      <input type="radio" name="languages" value="CSS"/>CSS
+      <input type="radio" name="languages" value="JavaScript"/>Javascript
```
Comment (codingcodymiller) [inline html-forms/index.html]: The text next to each of these should be a `label`

File: html-forms/index.html [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    </label>
+    <p>
+
+
+      Would you like like to sign up for more information?</p>
+    <label>
+      <input type="checkbox" name="Sign up" value="Sign me up!"/>Sign me up!
```
Comment (codingcodymiller) [inline html-forms/index.html:39]: "Sign me up!" should be a `label`

File: html-forms/index.html [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+    </label>
+
+
+    <div>Comments</div>
```
Comment (codingcodymiller) [inline html-forms/index.html]: This should be a `label`

File: html-forms/index.html [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+    <label>
+      <textarea name="comments" cols="20" rows="4"></textarea>
+    </label>
+    <div>
+      <label>
```
Comment (codingcodymiller) [inline html-forms/index.html]: There is no text labeling this button, so no `label` element is necessary

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
You should re-read the documentation on `label` elements 🤓

**Comment by patriciamanalang** [comment]
Screen Shot 2022-10-13 at 10 38 40 PM

File: html-forms/index.html [hunk 6]
```diff
@@ -1,54 +1,48 @@
...
-      <textarea name="comments" cols="20" rows="4"></textarea>
+    <label for="JavaScript">
+      <input type="radio" name="languages" value="JavaScript">Javascript
     </label>
+
+    <div class="signup">
+      <p>
+        Would you like like to sign up for more information?
+      </p>
+      <input type="checkbox" name="Sign up" value="Sign me up!">
```
Comment (codingcodymiller) [inline html-forms/index.html]: This needs an `id` attribute to pair with the `for` attribute on its `label`

File: html-forms/index.html [hunk 7]
```diff
@@ -1,54 +1,48 @@
...
-      <select name="Select an option">
-        <option value="Because it's interesting" selected="selected"> Because it's interesting</option>
-        <option value="Because it's fun" selected="selected">Because it's fun</option>
-        <option value="Because it's powerful" selected="selected">Because it's powerful</option>
-        <option value="Select an option" selected="selected">Select an option</option>
-      </select>
-    </label>
-
-
+    <select name="Select an option">
```
Comment (codingcodymiller) [inline html-forms/index.html]: This needs an `id` attribute to pair with the `for` attribute on its `label`

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**